wfd/common/message_handler.cpp

```diff
@@ -272,11 +289,28 @@ void MessageSenderBase::Handle(std::unique_ptr<Message> message) {
   if (!HandleReply(static_cast<Reply*>(message.get())))
     observer_->OnError(this);
 
+  sender_->ReleaseTimer(timeout_timer_);
+  timeout_timer_ = 0;
+
   if (cseq_queue_.empty()) {
     observer_->OnCompleted(this);
   }
 }
 
+bool MessageSenderBase::HandleTimeoutEvent(uint timer_id) const {
+  return timeout_timer_ == timer_id;
+}
+
+void MessageSenderBase::SetTimeout() {
+  uint timer_id = sender_->CreateTimer(TimeoutValue());
+  if (timer_id > 0)
+    timeout_timer_ = timer_id;
+}
+
+int MessageSenderBase::TimeoutValue() const {
```

Comment:
I think we do not need this method, KDefaultTimeoutValue can be used directly.
s/KDefaultTimeoutValue/kDefaultTimeoutValue -- coding style